source/developers/quickstarts/first_openedx_pr.rst

```diff
@@ -34,30 +34,35 @@ Before you start
    For the smoothest experience, we recommend that your computer has at least
    **16 GB of RAM, 2 CPUs, and at least 50 GB of Free disk space**.
 
-This tutorial is written for users of macOS with either an Intel or an ARM
-(Apple Silicon, M1, M2, etc.) processor or Linux (x86 or x86_64). Additionally,
+This tutorial is written for users of macOS or Linux.
+It has been tested with both common 64-bit processor families:
+x86-64 (a.k.a. Intel-64) and ARM-64 (used in newer Apple laptops, including M1, M2, etc.).
```

Comment:
I reworded this a bit to capture that:
* [tutor doesn't guarantee 32-bit support](https://docs.tutor.overhang.io/install.html)
* there's no reason the ARM-64 instructions shouldn't also work on Linux (🤞🏻) although I avoided explicitly saying it would work on such a system